src/briefcase/integrations/wix.py

```diff
@@ -29,10 +29,7 @@ def __init__(self, command, wix_home=None, bin_install=False):
             binaries-only install does not.
         """
         self.command = command
-        if wix_home:
-            self.wix_home = wix_home
-        else:
-            self.wix_home = command.tools_path / 'wix'
+        self.wix_home = wix_home or command.tools_path / 'wix'
```

Comment:
Logically correct, but I'd argue it's not at all obvious from reading what is going on here. The older if/else logic is a lot more explicit.